packages/common/src/api/track.ts

```diff
@@ -7,8 +7,22 @@ const trackApi = createApi({
   reducerPath: 'trackApi',
   endpoints: {
     getTrackById: {
-      fetch: async ({ id }: { id: ID }, { apiClient }) => {
-        return await apiClient.getTrack({ id })
+      fetch: async (
+        { id, currentUserId }: { id: ID; currentUserId?: ID },
+        { apiClient }
+      ) => {
+        return await apiClient.getTrack({ id, currentUserId })
+      },
+      fetchBatch: async (
+        { ids, currentUserId }: { ids: ID[]; currentUserId?: ID },
+        { apiClient }
+      ) => {
+        return (
+          (await apiClient.getTracks({
+            ids,
+            currentUserId: currentUserId ?? null
```

Comment:
weird that `getTrack` is ok with undefined but `getTracks` isn't

Reply:
Yeah not sure why the types are like this:
```
type GetTrackArgs = {
  id: ID
  currentUserId?: Nullable<ID>
  unlistedArgs?: {
    urlTitle: string
    handle: string
  }
  abortOnUnreachable?: boolean
}

type GetTracksArgs = {
  ids: ID[]
  currentUserId: Nullable<ID>
}
```

Maybe I should make them consistent. Also noticed that we aren't passing in `currentUserId` anywhere we do `useGetTrackById`, that might be fine if we don't need any personalized data